src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
Yeah I know what CIE xyz and CIE XYZ are two different things. Here is what I propose: when we mean XYZ, then we spell it XYZ. We continue spelling the rest (in particular the rgb part) in lower case to conform to our coding conventions.

The word "linear" is probably redundant but it makes it obvious that we're getting a linear value. It cannot hurt.

What do you think?